pkgs/tools/games/ukmm/default.nix

```diff
@@ -36,7 +37,12 @@ rustPlatform.buildRustPackage rec {
   ];
 
   preFixup = ''
-    gappsWrapperArgs+=(--prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libglvnd ]})
+    gappsWrapperArgs+=(--prefix LD_LIBRARY_PATH : ${
```

Comment:
LD_LIBRARY_PATH leaks into child processes (ie firefox if you click on a link). this can have undesirable effects. Can you instead:
- force-link the libs with NIX_LDFLAGS ?
- add them to the rpath with patchelf ?

Reply:
This took me a little bit longer than I expected, apparently `NIX_LDFLAGS` doesn't change any of the rust link flags. I noticed that the wrapper executable linked to the libraries from `NIX_LDFLAGS` but not `.ukmm-wrapped`... so I copied over a pattern I saw other packages were using with `RUSTFLAGS`.

It should be good now! Thanks for the review! :blush: